Cargo.toml

```
@@ -56,3 +56,6 @@ path = "examples/field_instances/main.rs"
 [[example]]
 name = "collectathon"
 path = "examples/collectathon/main.rs"
+
+[patch.crates-io]
+bevy_ecs_tilemap = { git = "https://github.com/rparrett/bevy_ecs_tilemap/", branch = "bevy13" }
```

This could be updated to `main` now
```suggestion
bevy_ecs_tilemap = { git = "https://github.com/rparrett/bevy_ecs_tilemap/", branch = "main" }
```